.github/workflows/near_crates_publish.yml

```diff
@@ -11,20 +13,29 @@ on:
 jobs:
   publish-cargo-crates:
     name: "Publish near-workspaces on crates.io https://crates.io/crates/near-workspaces"
-    runs-on: "ubuntu-22.04-16core"
+    runs-on: "ubuntu-20.04-16core"
```

Comment:
Is there a reason for reverting this to an older version? IIRC 20.04 is nearing EOL.

Reply:
It was unintentional. Maybe I checked out on an older version of the file.